File: loops/primes.py


def primes(start, end):
    rng = range(start, end + 1)
    primes = []

    for x in rng:
        if isPrime(x):
            primes.append(x)

    for c in primes:
        print(c)


def isPrime(n):
    if n > 1:
        for i in range(2, n):
            if n % i == 0:
                return False 
        else:
            return True 

    else:
        return False 


primes(25, 50)

File: loops/primes.js


const primes = (start, end) => {
    let result = [];

    for (let x = start; x <= end; x++) {
        if (isPrime(x)) {
            result.push(x);
        }
    }
    for (let a in result) {
        console.log(result[a]);
    }
}

const isPrime = num => {
    if (num > 1) {
        for (let y = 2; y < num; y++) {
            if (num % y == 0) {
                return false;
            }
        }
        return true;
    } else {
        return false;
    }
}


primes(25, 50);